test/parallel/test-timers-setInterval-excludes-callback-duration.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+const sleep = function(ms) {
+  const st = new Date().getTime();
+  while (new Date().getTime() < st + ms);
+};
+
+var cntr = 0;
```

Comment:
Linter will flag this and the next line and want it to be `let` instead of `var`.